stubs/urllib3/urllib3/util/retry.pyi

```diff
@@ -33,18 +53,21 @@ class Retry:
         allowed_methods=...,
         status_forcelist=...,
         backoff_factor=...,
+        backoff_max=...,
         raise_on_redirect=...,
         raise_on_status=...,
         history=...,
         respect_retry_after_header=...,
         remove_headers_on_redirect=...,
-        method_whitelist=...,
     ) -> None: ...
     def new(self, **kw): ...
     @classmethod
-    def from_int(cls, retries, redirect=..., default=...): ...
-    def get_backoff_time(self): ...
+    def from_int(cls, retries, redirect=..., default=...) -> "Retry": ...
```

Comment:
No need to have string-literal annotations in stubs; `from __future__ import annotations` behaviour is enabled by default 🙂 same applies for the `increment` method

```suggestion
    def from_int(cls, retries, redirect=..., default=...) -> Retry: ...
```

Reply:
BUT, on second thought, the annotation in the source code is wrong. You should annotate the method like this: add `from _typeshed import Self` at the top of the file, and then do:

```suggestion
    def from_int(cls: type[Self], retries, redirect=..., default=...) -> Self: ...
```